include/Ably/ARTRest+Private.h

```diff
@@ -0,0 +1 @@
+../../Source/ARTRest+Private.h
```

Comment:
Did you forget to remove the private headers from `include` directory? Since you've now added them to `PrivateHeaders/` already.

Reply:
I missed `ARTRest+Private.h` in `include/Ably` - it's removed now.
To be clear, there is no `Source/include` directory, now it's moved one level up and it's on the same level  as `Source`
changed id: https://github.com/ably/ably-cocoa/pull/1136/commits/d43ac41d14872a5c85600a24d5c2664591841b03

Reply:
There are more that you missed in `include/Ably`: https://github.com/ably/ably-cocoa/tree/feature/SPM-rc/include/Ably: `ARTPush+Private.h`, `ARTAuth+Private.h`

You've also exposed `ARTSRConstants.h`, `ARTSRURLUtilities.h`, `ARTSRSIMDHelpers.h`, `ARTSRRunLoopThread.h`, `ARTSRRandom.h`, `ARTSRProxyConnect.h`, `ARTSRPinningSecurityPolicy.h`, `ARTSRMutex.h`, `ARTSRLog.h`, `ARTSRIOConsumerPool.h`, `ARTSRIOConsumer.h`, `ARTSRHash.h`, `ARTSRHTTPConnectMessage.h`, `ARTSRError.h`, `ARTSRDelegateController.h`, which are **internal** headers of SocketRocket. 

I can also use symbols in this header file in my application (because they are available). Probably not a good idea. 😅